ScreenSharingExample/basicvideocall/ViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  ViewController.swift
+//  basicvideocall
+//
+//  Created by Dmitry Fedoseyev on 27.07.2021.
+//
+
+import UIKit
+import HMSSDK
+import ReplayKit
+
+class ViewController: UIViewController {
+    var hmsSDK = HMSSDK.build() { sdk in
+        sdk.appGroup = "group.live.100ms.screenshare-example"
+    }
+    
+    let pickerButton: RPSystemBroadcastPickerView = {
+        let picker = RPSystemBroadcastPickerView(frame: CGRect(origin: .zero, size: CGSize(width: 44, height: 44)))
+        picker.preferredExtension = "live.100ms.screenshare-example.BasicExtension"
+        return picker
+    }()
+
+    var trackViewMap = [HMSTrack: HMSVideoView]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        joinRoom()
+    }
+
+    func joinRoom() {
+        let config = HMSConfig(userName: "Pawan's iPhone", authToken: "inset token here")
```

Comment:
typo: insert. also maybe some abstract user name?)  you can add your name in the header instead